app/controllers/conference_controller.rb

```diff
@@ -21,10 +25,40 @@ def schedule
     conf_end = 20
     @conf_period = conf_end - @conf_start
 
-    if @dates == Date.current
-      @today = Date.current.strftime('%Y-%m-%d')
+    @day = Time.find_zone(@conference.timezone).today
+    # the schedule takes you to today if it is a date of the schedule
+    if @dates.include? @day
```

Comment:
Do we really need to calculate that in the controller?
@hennevogel what do you think?